### ISSUE TITLE
ParseException when single quote

### ISSUE THREAD

**Issue by DeanHere** [JSQLParser/JSqlParser#189]
Hi,

```
		String sql = "SELECT somefield as 'aa' FROM tblA";
		try {
			CCJSqlParser parser = new CCJSqlParser(new StringReader(sql));
			Statements parseStatements = parser.Statements();
```

Here got ParseException:

```
Encountered " <S_CHAR_LITERAL> "\'aa\' "" at line 1, column 21.
Was expecting one of:
    "DO" ...
```

**Comment by wumpz** [comment]
Look at #168. JSqlParser will not support single quotation of database object names or aliases.

**Comment by DeanHere** [comment]
Thanks for reply, I checked #168. But MySQL supports this, so just would like to confirm this that JSqlParser still will not support this?

```
mysql> select id as 'aa' from test.etl_source;
```

**Comment by wumpz** [comment]
Same discussion here #181. MySQL supports backticks, as JSqlParser does but no single quotes. There is a difference. Maybe you have a copy paste error.

**Comment by DeanHere** [comment]
OK, so the answer is there will no support for single quotes right. Will close this issue here.

**Comment by wumpz** [comment]
solved by #583
